### ISSUE TITLE
Featured images on full width pages

### ISSUE THREAD

**Issue by mrManner** [Scouterna/Scouterna-WPTheme#11]
Inte direkt en bugg, men jag tycker det ser knäppt ut såhär... Borde inte 
1. Bilden (givet att den är tillräckligt stor) täcka hela sidbredden, eller
2. Texten flöda runt bilden?

![image](https://f.cloud.github.com/assets/2606711/1306226/2257d3fe-31ac-11e3-8c4f-d07bb78a1b31.png)

**Comment by frozzare** [comment]
Detta går att lösa när man laddar upp en bild eller ändrar på en bild. Exempel: http://wp.forsmo.me/index.php/page-image-alignment/

När man laddar upp en bild finns en dropdown som heter "Alignment" där man väljer hur den ska flyta. 

![](http://wp.forsmo.me/media/bild1.png)

När man editerar en bild ser det ut såhär.

![](http://wp.forsmo.me/media/bild2.png)

Och då ser det ut såhär, inte jätte fint (finns css klasser som man kan styla så de blir mer avstånd mellan bilden och texten).

![](http://wp.forsmo.me/media/bild3.png)

**Comment by mrManner** [comment]
Kan inte se bilderna, Dropbox tycker inte jag får :(

Dropdownen finns inte när man använder utvald bild-funktionen, dock. För "vanligt" inlagda bilder funkar det helt ok, och utvalda bilder på normalbreda sidor ser helt ok ut. Det är just fallet med utvald bild och fullbredds-mallen som inte blir så lyckat. Ska kika på det någon gång, tänkte jag...

**Comment by sakjur** [comment]
Gör så, det verkar vara ont om marginaler på left-aligned bilder, går det att fixa?

**Comment by frozzare** [comment]
Fast vill man lägga bilden till höger eller vänster kan man lägga bilden i html editorn istället för i utvald bild? Jag har inte studerat vad utvald bild gör jämfört med att man lägger den i editorn.

(Fixade så bilderna inte länkas in från Dropbox i mitt inlägg ovan)

**Comment by sakjur** [comment]
Alltså, texten ligger PRECIS till höger om bilden, kan man lägga en padding på left-aligned-bilder? Hur hanterar WordPress dem?

**Comment by frozzare** [comment]
I detta fallet skriver nog WordPress bara ut bilden på en rad och texten på raden under. Det är klart man kan lägga in css som säger att bilderna alltid ska flyta till vänster och om bilden är tillräckligt bred så hamnar texten under bilden annars hamnar texten till höger om bilden. Fast om man ska göra så, borde man inte få välja någonstans om den ska flyta till höger eller vänster?

**Comment by mrManner** [comment]
Tanken (problemet?) med just utvalda bilder är att det inte går att göra några inställningar alls. De flesta teman jag sett använder bilden dels som thumbnail i olika listor, dels i ganska liten storlek på artikelsidan. Tycker inte nödvändigtvis det är ett problem att man inte kan välja -  utseendet blir konsekvent oavsett specifika skribenters dåliga smak, bilder av varierande storlek etc.

**Comment by frozzare** [comment]
I WordPress projekt där jag jobbar så använder vi utvalda bilden som fasta bilder, alltså att bilden bara passar på ett ställe och inte går att ändra storlek osv på. Behöver vi en till sån bild så lägger vi till ett till utvald bild fält. Vill man ha friheten till att editera bilden storlek och om den ska flyta så tycker jag personligen att det är sånt man får göra i editorn och de är nog så WordPress gänget har tänkt med. Den utvalda bilden i detta temat verkade inte göra något speciellt utan bara la sig innan texten och de kanske inte är rätt? Inlägg får ingen thumbnail bild vad jag kan se, man kanske borde göra så att den utvalda bilden är just en thumbnail bild och inte hamnar i innehållet?

**Comment by mrManner** [comment]
Precis, något sådant tror jag är bra. Alternativt ta bort utvald bild-fältet helt, och förlita oss på att användarna klarar av att lägga in bilder på det enkla sättet :)
